public_html/dashboard.css

```diff
@@ -19,6 +23,79 @@
 }
 
 li {
-	list-style-type: none;
+	list-style-type:none;
+}
+
+.navBarItem {
+	position:absolute;
+	width:100%;
+	height:20%;
+	left:0%;
+}
+
+.navBarInput {
+	type:'image';
+	alt:'Submit';
+	height:100%;
+	width:100%;
+}
+
+#homeItem {
+	top:0%;
+}
+
+#home {
+	src:url("/assets/hiveIcon.png");

```

Comment:
I'd personally say its ok to leave the image tags in the main file.  Just move all of the style components into the css.